src/zope/app/container/browser/contents.py

```diff
@@ -204,6 +205,11 @@ def renameObjects(self):
         ids = request.get("rename_ids")
         newids = request.get("new_value")
 
+        if isinstance(ids, six.string_types):
+            ids = [ids]
+        if isinstance(newids, six.string_types):
+            newids = [newids]
```

Comment:
That does sound like a BBB issue in zope.testbrowser...

I don't have the energy to dive into it now, so I'm going to ignore it. ;)